community/kernel/src/main/java/org/neo4j/kernel/impl/index/schema/NativeSchemaIndexAccessor.java

```diff
@@ -69,7 +69,7 @@
     public void drop() throws IOException
     {
         closeTree();
-        gbpTreeFileUtil.deleteFile( storeFile );
+        fileSystem.deleteFile( storeFile );
```

Comment:
Used to be `fileSystem.deleteFileOrThrow` underneath.

Reply:
oops, that moment when automatic refactoring didn't really work; fixed now